README.md

```diff
@@ -1,23 +1,1377 @@
+
+<br />
+
+#### 2.1 Download the `phoenix_live_view` Dependency
+
+Now that you've added the `phoenix_live_view` to `mix.exs`,
+you need to _download_ the dependencies.
+Run:
+
+```
+mix deps.get
+```
+
+You should see output similar to:
+
+```
+Resolving Hex dependencies...
+Dependency resolution completed:
+Unchanged:
+  cowboy 2.7.0
+  ...
+New:
+  phoenix_live_view 0.8.1
+* Getting phoenix_live_view (Hex package)
+```
+
+<br />
+
+### Step 3. Configure `signing_salt` in `config.exs`
+
+Phoenix LiveView uses a cryptographic salt
+to secure communications
+between client and server. 🔐 <br />
+You don't need to know what this is,
+just follow the instructions below and you'll be fine,
+but if you are curious,
+read: https://en.wikipedia.org/wiki/Salt_(cryptography)
+
+In your terminal run the following command:
+
+```
+mix phx.gen.secret 32
+```
+
+You should see output similar to the following:
+
+```
+iluKTpVJp8PgtRHYv1LSItNuQ1bLdR7c
+```
+
+> 💡 This is a **random** string generator
+that generates a **32 character string** of alphanumeric data, <br />
+so the result will be **different each time** you run the command.
+
+Copy the string into your computer's clipboard.
+
+
+Open your `config/config.exs` file
+and locate the line that begins with
+`live_view:`
+
+In this case it is the last line in the "Configures the endpoint" block:
+
+```elixir
+# Configures the endpoint
+config :live_view_counter, LiveViewCounterWeb.Endpoint,
+  url: [host: "localhost"],
+  secret_key_base: "K73oqZVIRIAck+a4sNK0V/hPujAYLeXGrKwax57JXFKMb8z64kgTaMF0Ys/Ikhrm",
+  render_errors: [view: LiveViewCounterWeb.ErrorView, accepts: ~w(html json)],
+  pubsub: [name: LiveViewCounter.PubSub, adapter: Phoenix.PubSub.PG2],
+  live_view: [signing_salt: "dUvMl2Sn"]
+```
+
+Replace the String value for `signing_salt`
+with the one you generated in your terminal:
+
+```elixir
+# Configures the endpoint
+config :live_view_counter, LiveViewCounterWeb.Endpoint,
+  url: [host: "localhost"],
+  secret_key_base: "K73oqZVIRIAck+a4sNK0V/hPujAYLeXGrKwax57JXFKMb8z64kgTaMF0Ys/Ikhrm",
+  render_errors: [view: LiveViewCounterWeb.ErrorView, accepts: ~w(html json)],
+  pubsub: [name: LiveViewCounter.PubSub, adapter: Phoenix.PubSub.PG2],
+  live_view: [signing_salt: "iluKTpVJp8PgtRHYv1LSItNuQ1bLdR7c"]
+```
+
+The _last_ line in the code block is the important one.
+
+
+> 🏁 At the end of Step 3 the
+[`config.exs`](https://github.com/dwyl/phoenix-liveview-counter-tutorial/blob/79d149c89655a6ddd452c93187e638b487aaf375/mix.exs#L33-L46)
+file should look like this:
+[config/config.exs#L16](https://github.com/dwyl/phoenix-liveview-counter-tutorial/blob/6b48e6e52a7e935985a473a8cd4b32d7bbcfff39/config/config.exs#L16)
+
+> 💡**Note**: in a _real world_ App,
+we would use an environment variable
+for the `signing_salt`
+to ensure it is kept secret.
+
+
+<br />
+
+### Step 4: Add `:fetch_live_flash` Plug to Browser Pipeline
+
+
+At the top of the file you should see the line
```

Comment:
We need to specify which file as we are switching from the config.exs file to the router.ex file
```suggestion
At the top of the [router.ex](https://github.com/dwyl/phoenix-liveview-counter-tutorial/blob/3a5e5dd73c325ef609203e597cd0f80ac06198f8/lib/live_view_counter_web/router.ex#L2) file  you should see the line
```

Reply:
Great point @SimonLab ✋